src/main/scala/com/gu/invoicing/common/ZuoraAuth.scala

```diff
@@ -46,28 +49,36 @@ object ZuoraAuth extends JsonSupport {
       )
       .asString
       .body
+      .tap { body =>
+        // although cloudwatch is pretty secure, it's a nice thing to redact if possible
+        val redactedBody = body.replaceAll(""""access_token":"[^"]*"""", """"access_token":"***REDACTED***"""")
+        logger.info(redactedBody)
+      }
       .pipe(read[AccessToken](_))
       .access_token
   }
-  private val timer = new Timer()
+  private val timer = new Timer(true)
   private def ZuoraOutageWarning(cause: Throwable) = new RuntimeException(
     """
       |Failed to authenticate with Zuora after multiple retries.
       |Possible zuora outage. Investigate ASAP! https://trust.zuora.com
       |""".stripMargin,
     cause,
   )
+
+  private val five_minutes: Int = 5 * 60 * 1000
+
   timer.schedule(
     new TimerTask {
       def run(): Unit = {
-        retry(getAccessToken()) // do not update cache on failure
+        retry(getAccessToken) // do not update cache on failure
           .foreach(token => accessToken = token)
       }
     },
-    0,
-    5 * 60 * 1000, // refresh token every 5 min
+    five_minutes, // wait 5 mins for first refresh
```

Comment:
I noticed it was doing a double auth on startup.
the attempt below happens on app startup, so the initial delay should have been 5 minutes.
original PR it's not mentioned and was probably not considered https://github.com/guardian/invoicing-api/pull/16